### ISSUE TITLE
<architecture> tag and excluding armv5/armv6 devices on Google Play

### ISSUE THREAD

**Issue by Tw1ddle** [openfl/lime-tools#96]
I want to make Android builds to distribute on Google Play that only target armv7 devices, but I found that by default a ```libs/armeabi``` folder is created instead of a ```libs/armeabi-v7a``` folder when building apks. This folder structure makes Google Play think that the app supports ```armv5/armv6``` devices when it doesn't. I cannot see a workaround for this on the Google Play dashboard, short of manually blacklisting from a list of 10000 devices (which would be way too boring). Unfortunately my apps are getting a fair number of 1-star reviews from people with these older/incompatible devices.

I believe that in issue #61 @jgranick explained the libs folders are set up this way because only having a single ```libs/armeabi-v7a``` folder with armv7-only apps was causing problems with some (armv7) devices. I wanted to know more about that issue if possible, so I know how severe it is and whether it would also affect "fat" builds that have both ```libs/armeabi``` and ```libs/armeabi-v7a``` folders? I'm mainly looking for clarification here, as fat builds would be an option for me if they don't cause many issues of their own. Thanks!

**Comment by maxracoongames** [comment]
Hi, I would like to know the issue of single libs/armeabi-v7a folder too. In my real experience (in 1 month ago), even though Armv5 and Armv6 devices are < 5%, they gave us bad review until my app reached 3.9 tier (from 4.3).

**Comment by jgranick** [comment]
Yep, in my Samsung Galaxy S (perhaps other devices) the applications would not run unless there was a "libs/armeabi" folder

If there are both folders, it prefers the armeabi-v7a directory, but otherwise failed to work. So that's why the tools use only a single folder if you target only armv7, and use both folders if you include armv5 and armv7 both.

I'm not sure of a good solution for this, I think the key point of friction here is the Google Play submission process, which should allow us to account for that.

**Comment by maxracoongames** [comment]
Thanks for the information. One question, I don't have Samsung Galaxy S, but Unity 4 puts only armeabi-v7a, so do all Unity 4 games can't launched there?

**Comment by Tw1ddle** [comment]
Thanks! I will just add support for armv5/armv6 devices too and hope for the best.

It would be interesting to know if Unity has a workaround for the libs folders problem on your device. I don't have a Samsung Galaxy S either.
